.github/workflows/auto_label.yml

```diff
@@ -0,0 +1,178 @@
+---
+name: Auto Label
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize]
+  issues:
+    types: [opened, edited]
+
+jobs:
+  auto-label-pr:
+    if: github.event_name == 'pull_request_target'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Auto-label from PR template
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prNumber = context.payload.pull_request.number;
+            const prBody = context.payload.pull_request.body || '';
+            const labels = [];
+
+            console.log(`Processing PR #${prNumber}`);
+            console.log(`PR body length: ${prBody.length}`);
+            console.log(`PR body preview: ${prBody.substring(0, 200)}...`);
+
+            // Map Type of Change checkboxes to labels
+            const typeLabels = {
```

Comment:
https://github.com/ansible/ansible-content-actions/blob/main/.github/workflows/check_label.yaml#L32 this should be more consistent with this. the entire effort is doubled it both workflows behave differently.